api-wrapper/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+clear
+
```

Comment:
You may add virtual environment activation check like:
```suggestion

if [ ! -d "venv" ]; then
    echo "Virtual environment not found. Run: python3 -m venv venv"
    exit 1
fi

if [ -z "$VIRTUAL_ENV" ]; then
    echo "Virtual environment not activated. Run: . venv/bin/activate"
    exit 1
fi

```